waddrmgr/scoped_manager.go

```diff
@@ -1326,6 +1342,74 @@ func (s *ScopedKeyManager) newAccount(ns walletdb.ReadWriteBucket,
 	return putLastAccount(ns, &s.scope, account)
 }
 
+// NewAccountWatchingOnly is similar to NewAccount, but for watch-only wallets.
+func (s *ScopedKeyManager) NewAccountWatchingOnly(ns walletdb.ReadWriteBucket, name string,
+	pubKey *hdkeychain.ExtendedKey) (uint32, error) {
+	if !s.rootManager.WatchOnly() {
```

Comment:
This should be removed, as otherwise it means that users are unable to import _additional_ keys into an existing full wallet which IMO is a compelling use case.

Reply:
Done.